api/tests/test_departmental_teams_api.py

```diff
@@ -0,0 +1,171 @@
+# Standard Library
+from unittest.mock import patch
+
+# Third-Party Imports
+from django.apps import apps
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+
+# App Imports
+from api.serializers import AssetSerializer
+from api.tests import APIBaseTestCase
+from api.tests.test_user import User
+from core.models import AllocationHistory, AssetStatus
+
+client = APIClient()
+
+
+class DepartmentTeamTestCase(APIBaseTestCase):
+    """
+    Test department teams api
+    """
+
+    def setUp(self):
+        self.departmental_team = apps.get_model(
+            "core", "DepartmentalTeam"
+        ).objects.create(name="Accounts", department=self.department)
+        self.departmental_team_1 = apps.get_model(
+            "core", "DepartmentalTeam"
+        ).objects.create(name="Travels", department=self.department)
+        self.departmental_teams_url = reverse("departmental-teams-list")
+        self.asset_assignee_team = apps.get_model(
+            "core", "AssetAssignee"
+        ).objects.create(team=self.departmental_team)
+
+    def test_non_authenticated_user_get_department_teams(self):
+        response = client.get(self.departmental_teams_url)
+        self.assertEqual(
+            response.data, {"detail": "Authentication credentials were not provided."}
+        )
+
+    @patch("api.authentication.auth.verify_id_token")
+    def test_authenticated_user_get_department_teams(self, mock_verify_token):
+        mock_verify_token.return_value = {"email": self.admin_user.email}
+        response = client.get(
+            self.departmental_teams_url,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user),
+        )
+        data = response.json()
+        # return types assertions
+        assert isinstance(data['results'], list)
```

Comment:
> This is pytest assertion style. Would this run if the test is run with the `python manage.py test` command?

Yes

Reply:
> Let's make the changes so that both `pytest` command and `python manage.py test` command works for all tests.

This has been refactored however to the unittest assertion  style